src/Microsoft.CodeAnalysis.Analyzers/Microsoft.CodeAnalysis.Analyzers.md

```diff
@@ -424,6 +424,30 @@ Prefer 'syntax.IsKind(kind)' to 'syntax.Kind() == kind' when checking syntax kin
 |CodeFix|True|
 ---
 
+## RS1035: Do not used APIs banned for analyzers
+
+The symbol has been marked as banned for use in analyzers, and an alternate should be used instead.
+
+|Item|Value|
+|-|-|
+|Category|MicrosoftCodeAnalysisCorrectness|
+|Enabled|True|
+|Severity|Error|
+|CodeFix|False|
+---
+
+## RS1036: Specify analyzer banned API enforcement setting
+
+A project containing analyzers or source generators should specify an analyzer banned API enforcement setting.
+
+|Item|Value|
+|-|-|
+|Category|MicrosoftCodeAnalysisCorrectness|
+|Enabled|True|
+|Severity|Error|
+|CodeFix|False|
```

Comment:
It would be awesome if this supported a code fix that either creates or appends to an .editorconfig file and adds `<%option%> = true` entry. Can you file a tracking issue for adding this code fix in future?

Reply:
#6133